apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V25__Add_partd_contract_number_index.sql

```diff
@@ -0,0 +1,59 @@
+
+create index ${logic.index-create-concurrently} "Beneficiaries_partd_contract_number_jan_id_idx"
+    on "Beneficiaries" ("partDContractNumberJanId")
+    ${logic.non-hsql-only-line} where length("partDContractNumberJanId") = 5
+    ;
```

Comment:
Did you want to keep the partial index, even though the `CONCURRENT` likely covers the original concern?

Reply:
I'm not seeing an appreciable relative improvement in performance with the partial index, so I'll plan on removing in the interest of simplicity.